traits/tests/test_trait_prefix_list.py

```diff
@@ -18,12 +18,12 @@
 from traits.api import HasTraits, TraitError, TraitPrefixList, Trait
 
 
-class A(HasTraits):
-    foo = Trait("one", TraitPrefixList("zero", "one", "two"))
-
-
 class TestTraitPrefixList(unittest.TestCase):
     def test_pickle_roundtrip(self):
+        with self.assertWarns(DeprecationWarning):
+            class A(HasTraits):
+                foo = Trait("one", TraitPrefixList("zero", "one", "two"))
+
```

Comment:
Ultra-nit: It would be nice if the assertion was in a separate test as those are two separate test objectives.